packages/client/src/util/toDataValue.ts

```diff
@@ -49,11 +44,6 @@ export async function toDataValue(
     return Promise.all(promiseArray);
   }
 
-  // attachments just send the rid directly
-  if (isAttachment(value)) {
```

Comment:
We don't need this check anymore as we just let users directly pass in the rid as a string (which we just pass through to the wire)